PokeAlarm/Filters/BaseFilter.py

```diff
@@ -57,12 +57,12 @@ def check_event(self, event):
 
     def reject(self, event, attr_name, value, required):
         """Log the reason for rejecting the Event."""
-        self._log.info("'%s' %s rejected by '%s'", event.name, self._type, self._name)
-        self._log.debug("%s incorrect: (%s to %s)", attr_name, value, required)
+        self._log.info(f"'{event.name}' {self._type} rejected by '{self._name}'")
+        self._log.debug(f"{attr_name} incorrect: ({value} to {required})")
```

Comment:
You revealed a good point. After some searches, it looks to be actually recommended to use %s formatting instead of F-Strings to not systematically evaluate strings. Especially when the 2/3 of the logs are at debug level and thus thrown away by default.

I made the changes and all above typos.